dotnet/src/SemanticKernel/Planning/KernelPlanningExtensions.cs

```diff
@@ -0,0 +1,85 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.SemanticKernel.Orchestration;
+
+namespace Microsoft.SemanticKernel.Planning;
```

Comment:
pls use the same namespace of `IKernel` to reduce the number of "using" statements. You can leave the code in this folder and add an exception if you like (see other extension methods)

Reply:
Bah editor undid this, fixing again.

Reply:
could you redo?

Reply:
> namespace Microsoft.SemanticKernel;